interactions/client/bot.pyi

```diff
@@ -51,12 +69,58 @@ class Client:
     async def wait_until_ready(self) -> None: ...
     def event(self, coro: Coroutine, name: Optional[str] = None) -> Callable[..., Any]: ...
     def change_presence(self, presence: ClientPresence) -> None: ...
+    @overload
+    async def get(
+        self, model: Type[Channel], http_method: Optional[str] = None, *, channel_id: Snowflake
+    ) -> Channel: ...
+    @overload
+    async def get(
+        self, model: Type[Guild], http_method: Optional[str] = None, *, guild_id: Snowflake
+    ) -> Guild: ...
+    @overload
+    async def get(
+        self,
+        model: Type[Member],
+        http_method: Optional[str] = None,
+        *,
+        guild_id: Snowflake,
+        member_id: Snowflake,
+    ) -> Member: ...
+    @overload
+    async def get(
+        self,
+        model: Type[Message],
+        http_method: Optional[str] = None,
+        *,
+        channel_id: Snowflake,
+        message_id: Snowflake,
+    ) -> Message: ...
+    @overload
+    async def get(
+        self,
+        model: Type[Role],
+        http_method: Optional[str] = None,
+        *,
+        guild_id: Snowflake,
+        role_id: Snowflake,
+    ) -> Role: ...
+    @overload
+    async def get(
+        self,
+        model: Type[User],
+        http_method: Optional[str] = None,
+        *,
+        user_id: Optional[Snowflake] = None,
+    ) -> User: ...
+    async def get(
+        self, model: T, http_method: Optional[str] = None, **kwargs: Dict[str, Snowflake]
+    ) -> T: ...
     def __check_command(
         self,
         command: ApplicationCommand,
         coro: Coroutine,
         regex: str = r"^[a-z0-9_-]{1,32}$",
-    )-> None: ...
+    ) -> None: ...
```

Comment:
```suggestion
```

I don't see how this is relevant to the PR.

Reply:
https://github.com/interactions-py/library/pull/702#discussion_r842232870